tests/check_spelling

```diff
@@ -1,10 +1,10 @@
 #! /bin/bash
 
 # exit fast if command is missing
-test -x $(which spellintian) || exit 0 ;
+test -x /usr/bin/spellintian || exit 0 ;
 
 THE_TEMP_FILE="/tmp/swapfile_spellcheck_${RANDOM}.tmp.txt" ;
-( (spellintian ${@:-./**/*} 2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") & (spellintian ${@:-./*} 2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") & (spellintian ${@:-./**/**/*}  2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") ) | sort -h | uniq | tee -a ${THE_TEMP_FILE:-/dev/null} ;
+( (spellintian "${@:-./**/*}" 2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") & (spellintian "${@:-./*}" 2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") & (spellintian "${@:-./**/**/*}"  2>/dev/null | fgrep -v "(duplicate word)" | fgrep " -> ") ) | sort -h | uniq | tee -a ${THE_TEMP_FILE:-/dev/null} ;
```

Comment:
Review: quotes help contain any path expansion (especially spaces) in a bash-shell friendly way.